install-monorepo.sh

```diff
@@ -22,33 +22,37 @@ fi
 OS="$(uname)"
 if [[ "$OS" == "Linux" ]]; then
 	ON_LINUX=1
+	if ! command -v apt &>/dev/null; then
+		abort "Installer script requires `apt` to ensure essentials are installed."
```

Comment:
There's no built in `abort` in bash, nor is it standard (at least on Linux).

You can define one though. Up near line 16, something like
```bash
function abort {
	echo "$*" >&2
        exit 1
}
```

Also, backtick inside double-quotes are apparently evaluated. You could backslash them or use single quotes instead.